worlds/generic/docs/setup_en.md

```diff
@@ -31,8 +34,8 @@ YAML is the file format which Archipelago uses in order to configure a player's
 game you will be playing as well as the settings you would like for that game.
 
 YAML is a format very similar to JSON however it is made to be more human-readable. If you are ever unsure of the
-validity of your YAML file you may check the file by uploading it to the check page on the Archipelago website. Check
-page: [YAML Validation Page](/mysterycheck)
+validity of your YAML file you may check the file by uploading it to the check page on the Archipelago website:
+[YAML Validation Page](/mysterycheck)
 
 ### Creating a YAML
```

Comment:
The first paragraph in this section could have the link at the end moved to attach to "the games page" in the first sentence.